src/app/shared/user/user.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="card" [ngStyle]="{'border': '1px solid greenyellow'}" *ngIf="route=='active-users' && !user.isDeleted">
```

Comment:
The component needs refactoring. The conditions for checking of routes can be removed and handled by the parent component which passes the data.

Reply:
did that by removing the routes